Packs/CommonScripts/Scripts/IsUrlPartOfDomain/IsUrlPartOfDomain.yml

```diff
@@ -0,0 +1,37 @@
+args:
+- default: false
+  description: Name of the domain
+  isArray: false
+  name: domain_name
```

Comment:
Needs to be able to handle a comma delimited domain list

Reply:
Done 👍

Reply:
@jochman please add test for that

Reply:
pushed